docs/user-guide/topics.md

```diff
@@ -0,0 +1,48 @@
+## Adding Topics to you Notebooks
+
+<a href="https://jovian.ml?utm_source=docs" target=_blank> Jovian </a> allows you to add topics to your notebooks. Adding topics to your notebooks helps other people find and contribute to your projects easily. You can add topics related to your project's intended purpose, subject area, frameworks/languages used, or other important qualities that you find useful.
+
+### About Topics
+
+With topics, you can explore notebooks in a particular subject area, find projects to contribute to, discover new solutions to a specific problem, or simply explore the frameworks that you love in action. Topics appear on the main page of a notebook. You can click a topic name to see related list of other notebooks tagged with that topic.
+
+<img src="https://i.imgur.com/HmBU5s0.png" class="screenshot" alt="Topics Box">
+
+To browse the most used topics, go to [https://jovian.ml/explore](https://jovian.ml/explore).
+
+### How to add Topics in your notebooks?
+
+1.&nbsp;Click on "Add Topics" button in the Notebook page.
```

Comment:
You can make these points bold, to highlight them in between images.
![image](https://user-images.githubusercontent.com/24864161/81444783-59bdff80-9195-11ea-8c48-f30e369e275b.png)